src/index.css

```diff
@@ -1,5 +1,6 @@
 html,
 body {
+	background-image: url(C:\Users\user\Desktop\tetra-cubes-game-1\src\images\TCStarBackground.gif);
```

Comment:
Should be relative path, not absolute because it would be deployed where different file system

Reply:
very true, was having issue there, should be good now